client/payment-details/timeline/map-events.js

```diff
@@ -223,6 +223,93 @@ const composeFXString = ( event ) => {
 	);
 };
 
+/**
+ * Returns an array containing fee breakdown.
+ *
+ * @param {Object} event Event object
+ *
+ * @return {Array} Array of formatted fee strings
+ */
+const feeBreakdown = ( event ) => {
+	if ( ! event?.fee_rates?.history ) {
+		return;
+	}
+
+	const {
+		fee_rates: { history },
+	} = event;
+
+	const feeLabelMapping = ( fixedRate ) => ( {
+		base:
+			0 !== fixedRate
+				? /* translators: %1$s% is the fee amount and %2$s is the fixed rate */
+				  __( 'Base fee: %1$s%% + %2$s', 'woocommerce-payments' )
+				: /* translators: %1$s% is the fee amount */
+				  __( 'Base fee: %1$s%%', 'woocommerce-payments' ),
+		'additional-international':
+			0 !== fixedRate
+				? __(
+						/* translators: %1$s% is the fee amount and %2$s is the fixed rate */
+						'International fee: %1$s%% + %2$s',
+						'woocommerce-payments'
+				  )
+				: __(
+						/* translators: %1$s% is the fee amount */
+						'International fee: %1$s%%',
```

Comment:
A suggestion for this label and the one above, should we be a little more explicit and change this:

"International card fee"

rather than

"International fee"

The reason is that "International fee" is a little ambiguous and might be confusing as we also have an FX fee, i.e. customers might think we're charging them twice for the same thing.

Reply:
Makes sense, fixed it in: https://github.com/Automattic/woocommerce-payments/pull/2500/commits/9ab0d164cfb93e2d434530b688b2cf9c70a2f805